static/template/_global/js/board_front.js

```diff
@@ -46,8 +46,11 @@ function addImages(board) {
     const specialRules = board.querySelectorAll("special-rules-container")[0];
     const spiritBorderSize = board.getAttribute("spirit-border-scale");
     const spiritNamePanel = board.querySelectorAll("spirit-name")[0];
+    console.log("here");
```

Comment:
@resonant-gamedesign One day I'm going to add a linting rule that disallows leaving in `console.log()` statements. Be prepared 😆